CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
 # CHANGELOG
 
+## Release 2.4.4 (in development)
+
+### Breaking changes
+- Due to a critical vulnerability with CVSS score of 10.0 that was discovered
+affecting all `Salt Master` versions inferior to `3000.1`, this release ships
+with a hot fix and of course a `Salt Master` version updated to `3000.3`.
+Users, especially those who expose the `Salt Master` to the internet must
+therefore upgrade immediately.
```

Comment:
I'm not sure this should land in `Breaking changes`, maybe add a `Security` section? WDYT @scality/metalk8s?